res/css/views/dialogs/security/_AccessSecretStorageDialog.scss

```diff
@@ -120,3 +117,28 @@ limitations under the License.
 .mx_AccessSecretStorageDialog_recoveryKeyEntry_fileInput {
     display: none;
 }
+
+.mx_AccessSecretStorageDialog_primaryContainer {
+    .mx_Dialog_buttons {
+        text-align: initial;
+        display: flex;
+        flex-flow: column;
+        gap: 14px;
+        padding-left: 24px; // See mx_AccessSecretStorageDialog_reset above
+
+        .mx_Dialog_buttons_additive {
+            float: none;
+        }
+
+        .mx_Dialog_buttons_items {
+            display: flex;
+            flex-wrap: wrap;
+            gap: 8px;
```

Comment:
We have variables for standard spacings, please use them here: https://github.com/matrix-org/matrix-react-sdk/blob/develop/res/css/_spacing.scss

Reply:
Please see: https://github.com/matrix-org/matrix-react-sdk/pull/8208#discussion_r840729631

Reply:
Spacing variables are the current best practice, and so we're requiring their use in all new code, even though very little old code has been updated to use them yet.

Reply:
I see, also there are some cases where spacing variables are not available, such as 3px/7px, etc.